qaseio/src/api.ts

```diff
@@ -0,0 +1,31 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Qase.io API
```

Comment:
please remove the same headers from everywhere

Reply:
Done